lib/rollbar/plugins/thread.rb

```diff
@@ -1,13 +1,14 @@
 Rollbar.plugins.define('thread') do
-  execute do
-    Thread.class_eval do
-      def initialize_with_rollbar(*args, &block)
+  module Rollbar
+    module ThreadPlugin
```

Comment:
I would have named this `Thread`, however, many other parts of the codebase use `Thread` instead of `::Thread`, so this causes a naming conflict. Rather than update all those files, I opted to choose a different name.